senic_hub/backend/views/nuimo_components.py

```
@@ -338,3 +359,25 @@ def test_blink_phue(component_ip, component_username, id):  # pragma: no cover
 
     except:
         return None
+
+
+def test_ring_sonos(component_ip):   # pragma: no cover
+    test_speaker = SoCo(component_ip)
+    uri_test = "http://files.senic.com/swblaster.mp3"
+    # TODO : Self-host the auido file on the hub & assign it's url to "uri_test", such as
+    # http://192.168.1.222:8000/swblaster.mp3
+    try:
+        test_speaker.volume = 10
+        test_speaker.play_uri(uri_test)
+        test_speaker.get_current_track_info()
+        time.sleep(2)
+        test_speaker.stop()
+        try:
+            test_speaker.play_from_queue(0)
+            test_speaker.stop()
+        except:
+            pass
```

Generally swallowing all exceptions is not a good practice. Either handle a specific exception by specifying the exception class or if you do a catch all, log it somewhere, so we can debug later.